src/Presentation/Nop.Web/Views/Checkout/Confirm.cshtml

```diff
@@ -73,6 +73,14 @@
                     <div class="buttons">
                         <input type="submit" name="nextstep" value="@T("Checkout.ConfirmButton")" class="button-1 confirm-order-next-step-button" />
                     </div>
+					<script>
+                        $(document).on('submit', 'form', function() {
+                            var button = $(this).find('input[type="submit"]');
+                            setTimeout(function() {

```

Comment:
But what will happen if some error happens? It won't be enabled back. So it can be clicked again. For example, payment error or "MinimumOrderPlacementInterval" setting requirement is not met